unidist/cli/backends/mpi/runner.py

```diff
@@ -109,7 +111,21 @@ def get_worker_command(num_cpus):
                 ),
             ]
 
-        command += [hosts_str] + command_executor + [":"] + command_monitor
+        if sys.platform == "win32":
+            import mpi4py
+
+            if mpi4py.get_config()["libraries"] == "impi":
+                command += hosts + command_executor + [":"] + command_monitor
+            else:
+                if len(self.hosts) == 1 and self.hosts[0] == get_localhost_ip():
```

Comment:
Is there an option for user not to pass any `host` parameter? Assuming that it will be a localhost.

Reply:
Sure, this is already implemented. This branch handles this case.